fuzzbucket/blueprints/boxes.py

```diff
@@ -1,39 +1,50 @@
 import typing
 
 import flask
+import flask_login
 
-from .. import auth, aws, box, cfg, datetime_ext, tags
+from .. import aws, box, cfg, datetime_ext, tags
 from ..log import log
 
 bp = flask.Blueprint("boxes", __name__)
 
 
 @bp.route("/", methods=("GET",), strict_slashes=False)
+@flask_login.login_required
 def list_boxes():
-    if not auth.is_fully_authd():
-        return auth.auth_403()
+    user_id: str | None = flask_login.current_user.get_id()
 
-    log.debug(f"handling list_boxes for user={flask.request.remote_user!r}")
+    if user_id is None:
```

Comment:
If login is required, can the method be entered without there being a user?

Reply:
Hopefully not, but I was feeling paranoid 😅

Reply:
I thought the decorator redirects to the login page if the user is not logged in.
Seems like a lot of protection code that will never be executed.

Reply:
That's how it's supposed to work 👍🏼  I'll dial back the paranoia 😅